browser/ipfs/ipfs_host_resolver_unittest.cc

```diff
@@ -196,32 +196,40 @@ TEST_F(IPFSHostResolverTest, SuccessOnReuse) {
   auto* network_context = GetNetworkContext();
   auto* fake_host_resolver_raw = fake_host_resolver.get();
   network_context->SetHostResolver(std::move(fake_host_resolver));
-  base::RunLoop run_loop;
   ipfs::IPFSHostResolver ipfs_resolver(network_context, prefix);
 
   SetResolvedCallbackCalled(0);
```

Comment:
This should be `SetResolvedCallbackCalled(false);` instead of 0/1, and same for everywhere in this file.